js/components/CreatePotluckProgressBar/styles.js

```diff
@@ -0,0 +1,32 @@
+import { StyleSheet } from 'react-native';
+
+import { colors, windowWidth as width, windowHeight as height } from '../../styles/baseStyles';
+
+const progressBarImageHeightWidthRatio = 59 / 702;
```

Comment:
They are the original image height and width, I can see if when we import the image it can be pulled off and stored.